kimchi/src/circuits/polynomials/permutation.rs

```diff
@@ -56,11 +56,16 @@ use rand::{CryptoRng, RngCore};
 
 /// Number of constraints produced by the argument.
 pub const CONSTRAINTS: u32 = 3;
-pub const ZK_ROWS: u64 = 3;
+/// Number of randomized rows (1 per evaluation).
+pub const ZK_ROWS: u64 = 2;
+/// Number of rows for the final accumulator value.
+pub const ACC_ROWS: u64 = 1;
+/// Total number of evaluation rows.
+pub const EVAL_ROWS: u64 = ZK_ROWS + ACC_ROWS;
```

Comment:
a few things here:

* `EVAL_ROWS` is not a good name, as it is the number of evaluation points, not evaluation rows. `EVALS` could be better. This is supposed to be set to 2 because we evaluate things at zeta, and zeta * omega (see the PLONK video series for an explanation on that)
* `EVALS` is not related to the permutation, it's also not related to the `circuits/` as it is a protocol value. Maybe a good idea to move it to `kimchi/src/lib.rs`

Reply:
(also, might be a good idea to just have these values be u32 instead of u64)